package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@govtechsg/sgds",
+  "name": "sgds-govtech",
```

Comment:
confirm is sgds-govtech and not @govtechsg/sgds-govtech ah ?

Reply:
yep confirm for now